crates/swc_ecma_minifier/tests/terser/compress/switch/issue_441_2/output.js

```diff
@@ -1 +1,2 @@
+foo;
```

Comment:
Bug of `Remover`. Yet another reason to have a minifier's own dead code remover, the one in optimization doesn't take into account of global vars